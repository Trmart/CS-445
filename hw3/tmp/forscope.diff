0a1,102
> .   .   .   .   Child: 0  Const true of type bool [line: 36]
> .   .   .   .   Child: 0  Const true of type bool [line: 45]
> .   .   .   .   Child: 0  Id: values [line: 10]
> .   .   .   .   Child: 0  Id: values [line: 15]
> .   .   .   .   Child: 0  Id: values [line: 22]
> .   .   .   .   Child: 0  Id: values [line: 28]
> .   .   .   .   Child: 0  Id: values [line: 3]
> .   .   .   .   Child: 0  Id: values [line: 35]
> .   .   .   .   Child: 0  Id: values [line: 44]
> .   .   .   .   Child: 0  Id: values [line: 5]
> .   .   .   .   .   .   Child: 0  Id: x [line: 12]
> .   .   .   .   .   .   Child: 0  Id: x [line: 19]
> .   .   .   .   .   Child: 0  Id: x [line: 23]
> .   .   .   .   .   .   Child: 0  Id: x [line: 25]
> .   .   .   .   .   .   Child: 0  Id: x [line: 30]
> .   .   .   .   .   .   .   Child: 0  Id: x [line: 31]
> .   .   .   .   .   .   Child: 0  Id: x [line: 37]
> .   .   .   .   .   .   .   Child: 0  Id: x [line: 38]
> .   .   .   .   .   .   .   Child: 0  Id: x [line: 41]
> .   .   .   .   .   .   Child: 0  Id: x [line: 46]
> .   .   .   .   .   .   .   Child: 0  Id: x [line: 48]
> .   .   .   .   .   Child: 0  Id: x [line: 7]
> .   .   .   .   Child: 0  Op: > [line: 23]
> .   .   .   Child: 0  Op: [ [line: 3]
> .   .   .   .   .   Child: 0  Op: > [line: 30]
> .   .   .   .   .   Child: 0  Op: > [line: 37]
> .   .   .   .   .   Child: 0  Op: > [line: 46]
> .   .   Child: 0  Var: values of array of type int [line: 2]
> .   .   .   Child: 0  Var: x of type int [line: 10]
> .   .   .   .   .   Child: 0  Var: x of type int [line: 11]
> .   .   .   Child: 0  Var: x of type int [line: 15]
> .   .   .   .   Child: 0  Var: x of type int [line: 16]
> .   .   .   .   .   Child: 0  Var: x of type int [line: 18]
> .   .   .   Child: 0  Var: x of type int [line: 22]
> .   .   .   .   .   Child: 0  Var: x of type int [line: 24]
> .   .   .   Child: 0  Var: x of type int [line: 28]
> .   .   .   .   Child: 0  Var: x of type int [line: 29]
> .   .   .   .   .   .   Child: 0  Var: x of type int [line: 31]
> .   .   .   Child: 0  Var: x of type int [line: 35]
> .   .   .   .   .   .   Child: 0  Var: x of type int [line: 38]
> .   .   .   .   .   .   Child: 0  Var: x of type int [line: 41]
> .   .   .   Child: 0  Var: x of type int [line: 44]
> .   .   .   .   .   .   Child: 0  Var: x of type int [line: 48]
> .   .   .   Child: 0  Var: x of type int [line: 5]
> .   .   .   .   Child: 0  Var: x of type int [line: 6]
> .   .   .   .   .   Child: 1  Assign: <= [line: 12]
> .   .   .   .   .   Child: 1  Assign: <= [line: 19]
> .   .   .   .   .   Child: 1  Assign: <= [line: 25]
> .   .   Child: 1  Assign: <= [line: 3]
> .   .   .   .   .   .   Child: 1  Assign: <= [line: 31]
> .   .   .   .   .   .   Child: 1  Assign: <= [line: 38]
> .   .   .   .   .   .   Child: 1  Assign: <= [line: 41]
> .   .   .   .   .   .   Child: 1  Assign: <= [line: 48]
> .   .   .   .   Child: 1  Assign: <= [line: 7]
> .   Child: 1  Compound [line: 1]
> .   .   .   .   Child: 1  Compound [line: 10]
> .   .   .   .   Child: 1  Compound [line: 17]
> .   .   .   .   Child: 1  Compound [line: 23]
> .   .   .   .   .   Child: 1  Compound [line: 30]
> .   .   .   .   .   Child: 1  Compound [line: 37]
> .   .   .   .   Child: 1  Const 0 of type int [line: 3]
> .   .   .   .   Child: 1  Const 10 of type int [line: 5]
> .   .   .   .   .   Child: 1  Const 1 of type int [line: 7]
> .   .   .   .   Child: 1  Const 20 of type int [line: 10]
> .   .   .   .   .   .   Child: 1  Const 2 of type int [line: 12]
> .   .   .   .   Child: 1  Const 30 of type int [line: 15]
> .   .   .   .   .   .   Child: 1  Const 3 of type int [line: 19]
> .   .   .   .   .   Child: 1  Const 3 of type int [line: 23]
> .   .   .   Child: 1  Const 3 of type int [line: 3]
> .   .   .   .   .   .   Child: 1  Const 3 of type int [line: 30]
> .   .   .   .   .   .   Child: 1  Const 3 of type int [line: 37]
> .   .   .   .   .   .   Child: 1  Const 3 of type int [line: 46]
> .   .   .   .   Child: 1  Const 40 of type int [line: 22]
> .   .   .   .   .   .   Child: 1  Const 4 of type int [line: 25]
> .   .   .   .   Child: 1  Const 50 of type int [line: 28]
> .   .   .   .   .   .   .   Child: 1  Const 5 of type int [line: 31]
> .   .   .   .   Child: 1  Const 60 of type int [line: 35]
> .   .   .   .   .   Child: 1  Const 666 of type int [line: 46]
> .   .   .   .   .   .   .   Child: 1  Const 6 of type int [line: 38]
> .   .   .   .   Child: 1  Const 70 of type int [line: 44]
> .   .   .   .   .   .   .   Child: 1  Const 7 of type int [line: 41]
> .   .   .   .   .   .   .   Child: 1  Const 7 of type int [line: 48]
> .   .   .   .   Child: 1  If [line: 30]
> .   .   .   .   Child: 1  If [line: 37]
> .   .   .   .   Child: 1  If [line: 46]
> .   .   .   Child: 1  Range [line: 10]
> .   .   .   Child: 1  Range [line: 15]
> .   .   .   Child: 1  Range [line: 22]
> .   .   .   Child: 1  Range [line: 28]
> .   .   .   Child: 1  Range [line: 35]
> .   .   .   Child: 1  Range [line: 44]
> .   .   .   Child: 1  Range [line: 5]
> .   .   .   Child: 2  Compound [line: 10]
> .   .   .   Child: 2  Compound [line: 15]
> .   .   .   Child: 2  Compound [line: 28]
> .   .   .   .   .   Child: 2  Compound [line: 40]
> .   .   .   .   .   Child: 2  Compound [line: 47]
> .   .   .   Child: 2  Compound [line: 5]
> .   .   .   Child: 2  If [line: 23]
> .   .   .   Child: 2  While [line: 36]
> .   .   .   Child: 2  While [line: 45]
> ERROR(11): Symbol 'x' is already declared at line 10.
1a104,105
> ERROR(18): Symbol 'x' is already declared at line 15.
> ERROR(24): Symbol 'x' is already declared at line 22.
2a107,109
> ERROR(31): Symbol 'x' is already declared at line 28.
> ERROR(38): Symbol 'x' is already declared at line 35.
> ERROR(48): Symbol 'x' is already declared at line 44.
4,5c111,120
< Number of errors: 3
< Number of warnings: 2
---
> Func: main returns type void [line: 1]
> Number of errors: 9
> Number of warnings: 9
> .   .   Sibling: 1  For [line: 5]
> .   .   Sibling: 2  For [line: 10]
> .   .   Sibling: 3  For [line: 15]
> .   .   Sibling: 4  For [line: 22]
> .   .   Sibling: 5  For [line: 28]
> .   .   Sibling: 6  For [line: 35]
> .   .   Sibling: 7  For [line: 44]
7a123,129
> WARNING(22): The variable 'x' seems not to be used.
> WARNING(28): The variable 'x' seems not to be used.
> WARNING(2): The variable 'values' seems not to be used.
> WARNING(35): The variable 'x' seems not to be used.
> WARNING(41): The variable 'x' seems not to be used.
> WARNING(44): The variable 'x' seems not to be used.
> WARNING(5): The variable 'x' seems not to be used.
