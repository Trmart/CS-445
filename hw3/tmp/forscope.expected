ERROR(16): Symbol 'x' is already declared at line 15.
ERROR(29): Symbol 'x' is already declared at line 28.
ERROR(6): Symbol 'x' is already declared at line 5.
Number of errors: 3
Number of warnings: 2
WARNING(10): The variable 'x' seems not to be used.
WARNING(15): The variable 'x' seems not to be used.
